repo/meta/index.json

```diff
@@ -22,6 +22,18 @@
       }
     },
     {
+      "currentVersion":"0.8.1-RC1",
+      "description":"A cluster-wide init and control system for services in cgroups or Docker containers.",
```

Comment:
I'd remove the "in cgroups or Docker containers." and say something like:
`A cluster-wide init and control system for containerized services."